### DISCUSSION TITLE
good morning

### DISCUSSION THREAD

**Discussion by omegajudith** [ersilia-os/ersilia#1283] [General] [answered]
hello team, my name is judith and glad to be here, please some help needed here.
![Screenshot from 2024-10-03 10-18-17](https://github.com/user-attachments/assets/4b0f567b-734e-416b-981e-a37581aeb0a9)
how do I open or fix a new bug?

**Comment by omegajudith** [comment]
do i need tom fill in all the above fields? please someone guide me

**Comment by DhanshreeA** [comment, marked as answer]
Hi @omegajudith please utilize Slack for any discussions and queries - we don't actively maintain or monitor these discussions. Thank you!
